lib/findJsModulesFor.js

```diff
@@ -101,7 +101,7 @@ function dedupeAndSort(modules: Array<JsModule>): Array<JsModule> {
     sorted,
     (module: JsModule): string => module.importPath,
   );
-  return sortBy(uniques, (module: JsModule): string => module.displayName());
+  return sortBy(uniques, (module: JsModule): string => module.importPath);
```

Comment:
At first, I was afraid this would change how we sort import statements, but after looking at how `displayName()` was implemented I believe this should be safe.